xlml/utils/maxtext.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import os
+from airflow.decorators import task
+
+
+@task
+def generate_maxtext_run_name(
+    run_name_prefix, curr_num_slices, tpu_version, tpu_cores, idx
+) -> str:
+  current_datetime = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
+  return f"{run_name_prefix}-{curr_num_slices}xv{tpu_version}-{tpu_cores}-{current_datetime}-{idx}"
```

Comment:
It looks like none of the logic here is specific to maxtext, could we rename the file and this method? Maybe something like `xlml/utils/name_format.py`?

Reply:
Done!